source/NVDAObjects/window/winword.py

```diff
@@ -776,6 +791,10 @@ def _normalizeFormatField(self,field,extraDetail=False):
 			else:
 				v=self.obj.getLocalizedMeasurementTextForPointSize(v)
 			field[x]=v
+		bullet=field.get('line-prefix')
+		if bullet and len(bullet)==1:
+			global mapPUAToUnicode

```

Comment:
On second thought, `global` is only necessary if you want to modify the value. Unnecessary use of this keyword was in the India code. Fixed.